pkg/openshift-sdn/informers.go

```diff
@@ -121,3 +125,31 @@ func applyClientConnectionOverrides(kubeConfig *rest.Config) {
 	kubeConfig.QPS = 10.0
 	kubeConfig.Burst = 20
 }
+
+type eventHandlingLogger struct{}
+
+func describe(obj interface{}) string {
+	if svc, ok := obj.(*v1.Service); ok {
+		if svc.Spec.Type == v1.ServiceTypeExternalName {
+			return fmt.Sprintf("externalname service %s/%s", svc.Namespace, svc.Name)
+		} else if svc.Spec.ClusterIP == v1.ClusterIPNone {
+			return fmt.Sprintf("headless service %s/%s", svc.Namespace, svc.Name)
+		} else {
+			return fmt.Sprintf("service %s/%s", svc.Namespace, svc.Name)
```

Comment:
What do you think about logging the entire object? We might realize later on that a couple of attributes are not enough to debug the issue completely?

Reply:
Sure, that makes sense. let me update the PR.

Reply:
yeah, agreed. I'd serialize the service object to json and log it.

Reply:
does it need serialization, I am logging it as `%+v`?